README.md

```diff
@@ -1 +1,142 @@
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 My Portfolio <a name="about-project"></a>
+
+Welcome to my portfolio project, where I showcase my skills and expertise! Here, you will find a collection of my best works, highlighting my abilities in Software Development. Whether you are a potential employer, colleague, or simply interested in my work, I invite you to check my portfolio and get to know me better. Thank you for visiting, and I look forward to connecting with you!
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+</details>
+
+<details>
+  <summary>CSS</summary>
+</details>
+
+<details>
+<summary>Git</summary>
+</details>
+
+<details>
+<summary>GitHub</summary>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Mobile Responsive**
+- **Modern and fun look**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://htmlpreview.github.io/?https://raw.githubusercontent.com/zam-cham/portfolio/toolbar-and-headline-section/index.html)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Currently the live demo is ment to be seen in the mobile view, once you open the live demo, right-click on the website and select "Inspect"
+
+### Prerequisites
+
+In order to run this project you need: A browser 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Cristian "ZamCham"**
+
+- GitHub: [@zam-cham](https://github.com/zam-cham)
+- Twitter: [@zamchamgames](https://twitter.com/zamchamgames)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/cristian-zamcham)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Mobile responsive**
+- [ ] **Loads Fast**
+- [ ] **User Friendly Design**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, let me know on any of my social media account ;)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse team
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] As mentioned by the previous reviewer, I think it would be better if you fix the broken `[MIT](./LICENSE)` link in your README file to make it more professional as it is still showing a **404** error page once clicked. And to fix that, you can only edit your `LICENSE.txt` file name to `LICENSE` only